docs/source/named_tensor.rst

```diff
@@ -204,9 +204,9 @@ and :meth:`~Tensor.reshape`, but have more semantic meaning to someone reading t
     >>> named_flat_imgs.names
     ('N', 'features')
 
-    >>> unflattened_imgs = imgs.view(32, 3, 128, 128)
-    >>> unflattened_named_imgs = named_flat_imgs.unflatten(
-            'features', [('C', 3), ('H', 128), ('W', 128)])
+    >>> unflattened_named_imgs = named_flat_imgs.unflatten('features', [('C', 3), ('H', 128), ('W', 128)])
+    >>> unflattened_named_imgs.names
+    ('N', 'C', 'H', 'W')
```

Comment:
What is wrong with the original code?

Reply:
Well three things (please open the link I provided in the description to see it for yourself):
1. the parameters of `unflatten` are rendered as the output, they are not - that's because they are moved to a new line that does not have a leading `>>>`
2. `unflattened_imgs` is not used, it's just misleading
3. the output of unflattened_named_imgs.names is not shown, unlike the `flatten` exactly above